modules/local/metadata_all.nf

```diff
@@ -22,7 +22,7 @@ process METADATA_ALL {
 	stdout
 
 	script:
-	if( !metadata_category ) {
+	if( metadata_category == "" ) {
```

Comment:
You can keep writing: `if( !metadata_category )`
empty string `== false`, otherwise ` == true`

Reply:
Totally true, reverted that below.